index.html

```diff
@@ -50,39 +49,64 @@ <h3>Furahia Insurance <br><span class="logo-writing2">Since 1965</span></h3>
       <h2 id="intro1">Enjoy life na Furahia Insurance!!</h2>
     </div>
   </header>
-  <section>
-    <article>    
-      <h1 id="intro2">PROMOTING A LIFE <br> FREE LIFESTYLE <br> 2022</h1>  
-      <p>Peace of mind comes from knowing you and your family are covered and protected by the best health insurance cover</p>
+  <main>
+    <article>
+      <h1 id="intro2">PROMOTING A LIFE <br> FREE LIFESTYLE <br> 2022</h1>
+      <p class="intro-words">Peace of mind comes from knowing you and your family are covered and protected by the best
+        health insurance cover</p>
       <h1 id="intro3">50 YEARS IN BUSINESS </h1>
       <P id="intro4">We advise, Insure and Invest</P>
     </article>
-    <section class="row products-categories">
+    <section class="products-categories">
       <h3 class="products1">Main Products</h3>
 
-      <div class="product-life">
-        <img class="product-img" src="./imgs/life.jpeg" alt="Life Insurance">
-        <h2 class="product-name">Life</h2>
-        <p class="product-description">Our job is to help you choose the right insurance cover for you so that you may Live Free!</p>
-      </div>
+      <div class="row all-products">
+        <div class="product-life col-3" id="Life">
+          <img class="product-img" src="./imgs/life.jpeg" alt="Life Insurance">
+          <h2 class="product-name">Life</h2>
+          <p class="product-description">Our job is to help you choose the right insurance cover for you so that you may
+            Live Free!</p>
+        </div>
+
+        <div class="product-life col-3" id="Health">
+          <img class="product-img" src="./imgs/health.jpeg" alt="Health Insurance">
+          <h2 class="product-name">Health</h2>
+          <p class="product-description">Your family’s health and safety means the world to you. We understand that and
+            want to help
+            you protect your loved ones from situations beyond your control.</p>
+        </div>
```

Comment:
- [ ] Kindly add two more programs to match the requirements of this section. Please try to follow the design given in order to achieve the tasks.
- [ ] Also, the button you placed here is meant to be on the mobile screen. On desktop screens, it should be a link with an underline and uppercase. Please remove the button style for desktop and maintain it for mobile screens. 👍 

**See your design**
![image](https://user-images.githubusercontent.com/50941074/169515192-79b47259-e73b-416a-8da1-19abd83b0df9.png)

**The original design**
![image](https://user-images.githubusercontent.com/50941074/169515323-8b69cadb-a47d-492a-afba-5d1591ee5a93.png)

Reply:
@AmaduKamara This has been implemented as suggested